cloudflare/resource_cloudflare_device_posture_rule_test.go

```diff
@@ -32,11 +32,58 @@ func TestAccCloudflareDevicePostureRuleBasic(t *testing.T) {
 					resource.TestCheckResourceAttr(name, "input.0.id", "asdf-123"),
 				),
 			},
+			{
+				Config: testAccCloudflareDevicePostureRuleConfigOsVersion(rnd, accountID),
```

Comment:
can you please split these into individual tests? multiple assertions per test are very difficult to debug if something isn't correct in one of them but not the others.

Reply:
["steps"](https://www.terraform.io/docs/extend/testing/acceptance-tests/teststep.html#steps) in Terraform acceptance tests are kind of like different states to test against. example is you create a resource and then update it N times to confirm the state.

Reply:
I will make this change.